modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase

```diff
@@ -221,6 +221,50 @@ definition {
 		}
 	}
 
+	@description = "LPS-155308. Assert one filter can be removed."
+	@priority = "4"
+	test OneFilterCanBeRemoved {
+		task ("When there are several filters already established") {
+			FDSFilters.openFilters();
+
+			FDSFilters.addDateRangeFilters(
+				key_fromDate = "01/01/1901",
+				key_toDate = "02/02/1901");
+
```

Comment:
For some reason when editing the dates in Poshi (works fine manually), it only modifies the last number of the year. So when adding the filter it defaults to 1901. I thought it might be related to the date filter not working the sample portlet [LPS-156836](https://issues.liferay.com/browse/LPS-156836). 

![dates](https://user-images.githubusercontent.com/28836926/177436049-0534a009-c63d-4c53-b896-21386f07ec43.png)

Reply:
Or else I would've picked 2021, but I thought it was fine for now because we are just checking multiple filters can be removed one by one in this test.

Reply:
Fixed by replacing `Type` with `Type.sendKeys`. See changes in https://github.com/liferay-frontend/liferay-portal/pull/2486/commits/9a8adda88351159d5e881196a3ed630c5254a4a9

Reply:
:open_mouth: nice! thanks!